otj-server-core/src/main/java/com/opentable/server/PortSelector.java

```diff
@@ -40,19 +44,24 @@
         NOT_FOUND
     }
 
+    private final Environment environment;
+    private int portIndex = 0;
```

Comment:
should this be an atomic integer. Guess I'll see

Reply:
Looks like it technically should be, as there's no guarantee of same thread ALWAYS